conf/defaults.ini

```diff
@@ -1360,7 +1360,12 @@ execute_alerts = true
 evaluation_timeout = 30s
 
 # Number of times we'll attempt to evaluate an alert rule before giving up on that evaluation. The default value is 3.
+# The retry mechanism will stop if this number is reached or if the rule's evaluation interval is exceeded.
 max_attempts = 3
+# The initial delay before retrying a failed alert evaluation. This is the starting point for exponential backoff.
+initial_retry_delay = 1s
+# The maximum delay between retries during exponential backoff. Once this delay is reached, further retries will use this interval.
+max_retry_delay = 1s
```

Comment:
Both settings are `1s` to preserve the current behaviour of fixed `1s` delay between the retries.